util/dashboard/gen_dashboard_entry.py

```diff
@@ -102,8 +105,8 @@ def get_linked_checklist(obj, rev, stage, is_latest_rev=True):
 
     url = ""
     in_page_ref = ""
-    if rev[stage] not in ["D0", "V0"]:
-        # if in D0 or V0 stage, there is no in-page reference.
+    # if N/A or in D0/V0 stage, there is no in-page reference.
+    if rev[stage] not in ["D0", "V0", "N/A"]:
```

Comment:
Wonder if this is better:
```python
    if rev[stage] not in STAGE_STRINGS.keys():
```

Reply:
Hm... that would test for all keys, right? It is supposed to check for a subset only ("D0", "V0", "N/A")...